working-groups/diagnostics/README.md

```diff
@@ -0,0 +1,43 @@
+# Diagnostics Working Group
+![working group status: active][status]
+
+This working group aims to make rustc better at telling the user why the compiler isn't smart enough
+yet to understand their code.
+
+- **Leads:** [@oli-obk][oli-obk] [@estebank][estebank]
+- **Meeting Notes:** [All](NOTES.md)
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Make diagnostics emitting independent of the happy code path
+- Separate the diagnostics rendering from the compiler so it can be reused by others.
```

Comment:
Double checking: does these two points encompass making the happy path code cleaner?

Reply:
yes